extensions/ql-vscode/test/unit-tests/databases/db-manager.test.ts

```diff
@@ -362,6 +369,137 @@ describe("db manager", () => {
     });
   });
 
+  describe("expanded behaviours", () => {
+    it("should add item to expanded state", async () => {
+      // Add item to config
+      const listName = "my-list-1";
+      const dbConfig = createDbConfig({
+        remoteLists: [{ name: listName, repositories: [] }],
+      });
+
+      await saveDbConfig(dbConfig);
+
+      // Add item to expanded state
+      const dbItem = createVariantAnalysisUserDefinedListDbItem({
+        listName,
+      });
+
+      await dbManager.addDbItemToExpandedState(dbItem);
+      const expandedItems = await app.workspaceState.get<ExpandedDbItem[]>(
```

Comment:
Does this need the `await`? I was under the impression it's a synchronous operation.

Reply:
I just checked and indeed this is synchronous, so we can remove the `await`.